dll/shellext/fontext/fontpidl.cpp

```diff
@@ -0,0 +1,38 @@
+/*
+ * PROJECT:     ReactOS Font Shell Extension
+ * LICENSE:     GPL-2.0-or-later (https://spdx.org/licenses/GPL-2.0-or-later)
+ * PURPOSE:     pidl handling
+ * COPYRIGHT:   Copyright 2019 Mark Jansen (mark.jansen@reactos.org)
+ */
+
+#include "precomp.h"
+
+LPITEMIDLIST _ILCreate(LPCWSTR lpString, ULONG Index)
+{
+    // Because the FontPidlEntry contains one WCHAR, we do not need to take the null terminator into account
+    int cbData = sizeof(FontPidlEntry) + wcslen(lpString) * sizeof(WCHAR);
+    FontPidlEntry* pidl = (FontPidlEntry*)CoTaskMemAlloc(cbData + sizeof(WORD));
+    if (!pidl)
+        return NULL;
+
+    ZeroMemory(pidl, cbData + sizeof(WORD));
+
+    pidl->cb = cbData;
+    pidl->Magic = 'fp';
```

Comment:
Don't use multiple characters literal.

Reply:
Why?

Reply:
Their behavior is implementation-defined: https://stackoverflow.com/questions/3960954/c-multicharacter-literal

Reply:
Which is relevant for cross-platform serialization, but it is both created and checked from the same compiler so it should be fine. (And we do this in more places, f.e. pool tags)